CRM/Utils/System/Drupal8.php

```diff
@@ -65,7 +65,7 @@ public function createUser(&$params, $mail) {
     if ($user_register_conf != 'visitors' && !$user->hasPermission('administer users')) {
       $account->block();
     }
-    elseif (!$verify_mail_conf) {
+    elseif ($verify_mail_conf) {
```

Comment:
Can you add 
```patch
- elseif ($user->hasPermission('administer users') || $verify_mail_conf) {
+ elseif ($verify_mail_conf) {
```

Reply:
This would be so that account creation from CiviCRM would not block accounts by default?

Reply:
@mlutfy @monishdeb  what is the status here ? It seems pretty thoroughly stalled so unless you can agree it should be merged I think we should close & track through gitlab